openapi2conv/openapi2_conv.go

```diff
@@ -789,19 +844,19 @@ func FromV3SchemaRef(schema *openapi3.SchemaRef, components *openapi3.Components
 				}
 			}
 			return nil, &openapi2.Parameter{
-				In:              "formData",
-				Name:            originalName,
-				Description:     schema.Value.Description,
-				Type:            paramType,
-				Enum:            schema.Value.Enum,
-				Minimum:         schema.Value.Min,
-				Maximum:         schema.Value.Max,
-				ExclusiveMin:    schema.Value.ExclusiveMin,
-				ExclusiveMax:    schema.Value.ExclusiveMax,
-				MinLength:       schema.Value.MinLength,
-				MaxLength:       schema.Value.MaxLength,
-				Default:         schema.Value.Default,
-				Items:           schema.Value.Items,
+				In:           "formData",
+				Name:         originalName,
+				Description:  schema.Value.Description,
+				Type:         paramType,
+				Enum:         schema.Value.Enum,
+				Minimum:      schema.Value.Min,
+				Maximum:      schema.Value.Max,
+				ExclusiveMin: schema.Value.ExclusiveMin,
+				ExclusiveMax: schema.Value.ExclusiveMax,
+				MinLength:    schema.Value.MinLength,
+				MaxLength:    schema.Value.MaxLength,
+				Default:      schema.Value.Default,
+				// Items:           schema.Value.Items,
```

Comment:
I have commented this out as Items is only required when type is "array". Since this is a binary/file format type. Array is not required. 

https://spec.openapis.org/oas/v3.0.3#schema-object